### ISSUE TITLE
twitter feeds display only 20 

### ISSUE THREAD

**Issue by petrospap** [christianvuerings/jquery-lifestream#202]
Hello, in lifestream twitter display only 20 feeds, is there any case to increase twitter feeds and display them?
Also (and for me is very important!!!)
Can anyone give me any example how to display on tweets the time that have posted?
Please I need badly this one but I don’t know how to make it, any idea?

**Comment by christianvuerings** [comment]
@petrospap indeed, this is because we're scraping them from their website instead of using the API (since we don't want people to have to do the oAuth process). If you want +20, you should try updating the twittery proxy: https://github.com/christianvuerings/twittery/blob/master/server.js

**Comment by petrospap** [comment]
Thank you christianvuerings, 
do i have to get https://github.com/cheeriojs/cheerio ?
what is the "express" and "request"
can you give any hint how to use server.js with lifestream?
